Userland/Applications/Browser/Tab.cpp

```diff
@@ -407,10 +407,12 @@ Optional<URL> Tab::url_from_location_bar(MayAppendTLD may_append_tld)
     String text = m_location_box->text();
 
     StringBuilder builder;
-    builder.append(text);
+
+    String textUrl = g_search_engine.replace("{}", text);
```

Comment:
"Use snake_case (all lowercase, with underscores separating words) for variable and function names."
See CodingStyle.md